### ISSUE TITLE
When I use blazor.server,I can't use AppService

### ISSUE THREAD

**Issue by guolianyu** [abpframework/abp#8701]
Version: 4.3.0-rc.2

![image](https://user-images.githubusercontent.com/29138525/115365237-c4c1e400-a1f6-11eb-9d90-5023ee97873f.png)

**Comment by realLiangshiwei** [comment]
* Steps needed to **reproduce** the problem.

**Comment by emilsjog** [comment]
Version 4.3
Tested to migrate from WASM to blazor-server and noticed an error similar to this one. 
I have an interface `IEventsAppService : IApplicationService` and the error occurs if the service implementing the interface is called `EventAppService : IEventsAppService`. I get no error and it works if i use `EventsAppService : IEventsAppService`. Notice that with the S in the name of the implementing service I get no error.
Not sure if this is only related to blazor-server or ABP version 4.3. Had no issues on version 4.2.1 that I used before
